pkg/config/expand_test.go

```diff
@@ -261,7 +261,6 @@ func (s *MySuite) TestCombineLabels(c *C) {
 
 	coral := Module{
 		Source: "blue/salmon",
-		Kind:   TerraformKind,
```

Comment:
Yes, but `Kind` is irrelevant in context of this test, so I decided to keep it short.

Reply:
> I'm concerned we may no longer be testing the Kind of any real module.

I read you concern this way: 
* "The whole execution tree of `combineLabels` is independent of `Kind`"  - can be a too strong of assertion to make, it can be a mistake, adds needles extra mental load;
* `combineLabels` can became dependent on `Kind` in the future, without us realizing that assertion above is broken.

Reply:
I'm willing to accept that UnknownKind - in this instance - tests desired behavior because in essentially every case it will generate an error. That it does not within this stack is a sign that Kind is irrelevant. If we encounter problems later, I may ask that we loop over all Kinds here.